src/CommunicationApi/Services/Blobstorage/BlobImageMediaServiceProvider.cs

```diff
@@ -64,7 +65,7 @@ public async Task PersistMediaFile(UserInfo userInfo, string mediaUrl)
             await blobClient.UploadAsync(mediaStream, null,
                 new Dictionary<string, string>
                 {
-                    {"user", userInfo.Name}
+                    {"user", Convert.ToBase64String(Encoding.UTF8.GetBytes(userInfo.Name))}
```

Comment:
@renebik  : why are you storing this as base64 ?

Reply:
When I used my name. i.e. 'René', I got an error stating 'Request headers must contain only ASCII characters.', So I added this to overcome this.

Reply:
and are you somewhere Converting back from Base64, when returning the image in the mediaserviceprovider?  Because I don't think I've seen that.
You'd have to do this at the following line:

https://github.com/CoditEU/htc-social-connect/blob/fca50daa8cafa8543909f66528e7cb6cc3fa186e/src/CommunicationApi/Services/Blobstorage/BlobImageMediaServiceProvider.cs#L132

Reply:
Good catch, did not see that.